tests/integration/users.test.js

```diff
@@ -1463,7 +1422,24 @@ describe('PUT /users/:id/certificates', () => {
       expect(response.statusCode).toBe(200);
     });
 
-    const roles = [{ name: 'helper', expectedCode: 403 }, { name: 'planning_referent', expectedCode: 403 }];
+    it('should update the user by removing certificate, even if user is not from same company', async () => {
+      authToken = await getToken('training_organisation_manager');
+
+      const res = await app.inject({
+        method: 'PUT',
+        url: `/users/${auxiliaryFromOtherCompany._id.toHexString()}/certificates`,
+        payload: updatePayload,
+        headers: { Cookie: `alenvi_token=${authToken}` },
+      });
+
+      expect(res.statusCode).toBe(200);
+    });
```

Comment:
pardon si ce n'etait pas clair - je le mettrai plutot directement dans 
```
const roles = [
      { name: 'helper', expectedCode: 403 },
      { name: 'planning_referent', expectedCode: 403 },
      { name: 'trainer', expectedCode: 403 },
];
```
avec un ligne en 200 pour le TOM

la je trouve que ca porte a confusion car le it laisse penser que tout le monde peut modifier pour un autre